src/components/SEO/MetaTags.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import PropTypes from "prop-types"
+import Helmet from "react-helmet"
+
+import Keywords from "./Keywords"
+import OpenGraph from "./OpenGraph"
+import Twitter from "./Twitter"
+
+const MetaTags = ({
+  description,
+  image,
+  lang,
+  keywords,
+  title,
+  twitter,
+  url,
+}) => (
+  <>
+    <Helmet htmlAttributes={{ lang }} title={title}>
```

Comment:
Why not just put everything into a single component, splitting this off is just creating entropy.